src/bikeracks/__init__.py

```diff
@@ -56,8 +56,8 @@ def create_app(test_config=None):
     if not app.debug:
         if not os.path.exists('instance/logs'):
             os.mkdir('instance/logs')
-        file_handler = RotatingFileHandler('instance/logs/errors.log', maxBytes=10240,
-            backupCount=10)
+        file_handler = RotatingFileHandler('instance/logs/errors.log', maxBytes=512000,
```

Comment:
1k = 1024, not 1000. 512k is 512 * 1024 == 524288.

It's not important for this particular thing since the log size is arbitrary anyway, but it's important to keep that distinction in your head in general or you'll be confused when trying to get numbers to line up.